webapp/components/signup/signup_controller.jsx

```diff
@@ -79,14 +79,14 @@ export default class SignupController extends React.Component {
                             }
                         );
                     },
-                    () => {
+                    (e) => {
                         this.setState({ // eslint-disable-line react/no-did-mount-set-state
                             noOpenServerError: true,
                             loading: false,
                             serverError: (
                                 <FormattedMessage
-                                    id='signup_user_completed.invalid_invite'
-                                    defaultMessage='The invite link was invalid.  Please speak with your Administrator to receive an invitation.'
+                                    id={e.id}
+                                    defaultMessage={e.message}
```

Comment:
I think it does it for the Spanish.

![selection_009](https://cloud.githubusercontent.com/assets/15072209/19670133/7e6fa3c2-9a97-11e6-9c89-53a95b34dc77.png)